configure.ac

```diff
@@ -268,6 +269,11 @@ case "$host" in
                 	mono_cv_uscore=no
                 fi
 		case "$host" in
+		*-tizen-linux-*)
+			platform_tizen=yes
+			;;
+		esac
+		case "$host" in
```

Comment:
can't you remove those two lines?

Reply:
@lewurm  I could. The original case if for architecture checks where this is a (sub)platform check. I wasn't sure if they should be merged or not. For example, if x86 is added to original check I likely want whatever is done there performed for Tizen as well.

Reply:
@joncham ah I see, that makes sense 👍